### PR TITLE
Update badge links

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [![Build Status](https://travis-ci.com/pulumi/pulumi-aws.svg?token=eHg7Zp5zdDDJfTjY8ejq&branch=master)](https://travis-ci.com/pulumi/pulumi-aws)
 [![Slack](http://www.pulumi.com/images/docs/badges/slack.svg)](https://slack.pulumi.com)
-[![NPM version](https://badge.fury.io/js/%40pulumi%2Fpulumi.svg)](https://npmjs.com/package/@pulumi/pulumi)
-[![Python version](https://badge.fury.io/py/pulumi.svg)](https://pypi.org/project/pulumi)
-[![GoDoc](https://godoc.org/github.com/pulumi/pulumi?status.svg)](https://godoc.org/github.com/pulumi/pulumi)
-[![License](https://img.shields.io/npm/l/%40pulumi%2Fpulumi.svg)](https://github.com/pulumi/pulumi/blob/master/LICENSE)
+[![NPM version](https://badge.fury.io/js/%40pulumi%2Fpulumi.svg)](https://npmjs.com/package/@pulumi/aws)
+[![Python version](https://badge.fury.io/py/pulumi.svg)](https://pypi.org/project/pulumi-aws)
+[![GoDoc](https://godoc.org/github.com/pulumi/pulumi?status.svg)](https://godoc.org/github.com/pulumi/pulumi-aws)
+[![License](https://img.shields.io/npm/l/%40pulumi%2Fpulumi.svg)](https://github.com/pulumi/pulumi-aws/blob/master/LICENSE)
```
Comment (lukehoban) [inline README.md]: I think all the svgs still point to the wrong place?

Comment (stack72) [reply README.md]: All fixed now :) Thanks to @bermudezmt

Comment (lukehoban) [reply README.md]: FWIW - I think the license svg is still wrong. Probably doesn’t matter much as we use the same license across these.

**Comment by stack72** [comment]
This now looks much better - so going to merge this and have it available for the launch :)

**Pull request merged**